src/analytics/redux/tracked_events.js

```diff
@@ -5,6 +5,56 @@
 import { select } from '@wordpress/data';
 import { sendEventToHost } from '@wordpress/react-native-bridge';
 
+const INSERTERS = {
+	HEADER_INSERTER: 'header-inserter',
+	SLASH_INSERTER: 'slash-inserter',
+	QUICK_INSERTER: 'quick-inserter',
+};
+
+/**
+ * Guess which inserter was used to insert/replace blocks.
+ *
+ * @param {string[]|string} originalBlockIds ids or blocks that are being replaced
```

Comment:
Nitpick: Since we can pass a single value, we could reflect this in the argument description.
```suggestion
 * @param {string[]|string} originalBlockIds ID(s) of block(s) that are being inserted/replaced
```

Reply:
Updated in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5976/commits/b2e4ee34b0b97710deb3dbf3aca02d306da753f8